bindings/javascript/CMakeLists.txt

```diff
@@ -30,8 +30,9 @@ if (WHISPER_WASM_SINGLE_FILE)
 endif()
 
 set_target_properties(${TARGET} PROPERTIES LINK_FLAGS " \
-    --bind \
+    -lembind \
```

Comment:
`--bind` seems to be deprecated in favor of `-lembind`.

https://emscripten.org/docs/tools_reference/emcc.html#emcc-bind

<img width="570" alt="Screenshot 2022-12-27 at 01 04 51" src="https://user-images.githubusercontent.com/1218868/209589422-504cc017-9557-49f5-bee2-e35e35e117c6.png">